exo-go/src/applicationdeployer/deployer.go

```diff
@@ -1,18 +1,18 @@
-package application
+package applicationdeployer
```

Comment:
Thoughts on shortening this to `"appdeployer"`? Same for `"applicationrunner"` and `"applicationtester"`.